src/apis/emscripten/utils.rs

```diff
@@ -1,16 +1,16 @@
-use crate::webassembly::module::Module;
-use crate::webassembly::Instance;
+use crate::runtime::{Instance, Module};
+//use crate::webassembly::Instance;
 use byteorder::{ByteOrder, LittleEndian};
 use libc::stat;
 use std::ffi::CStr;
 use std::mem::size_of;
 use std::os::raw::c_char;
 use std::slice;
-
+use std::sync::Arc;
 /// We check if a provided module is an Emscripten generated one
-pub fn is_emscripten_module(module: &Module) -> bool {
```

Comment:
Any reason why you changed this to take an `Arc` instead of a ref?

Reply:
I've fixed this, changed it back to `&Module`.